test-data/unit/stubgen.test

```diff
@@ -595,6 +606,17 @@ class X(NamedTuple):
     a: Incomplete
     b: Incomplete
 
+[case testNamedtupleWithTypes]
+import collections, x
+X = typing.NamedTuple('X', [('a', str), ('b', str)])
```

Comment:
Can you please also test a case like `X = typing.NamedTuple('X', [('@', int)])`?

Reply:
@sobolevn, won't that fail at runtime?

Reply:
I will fail at runtime:
E   ValueError: Type names and field names must be valid identifiers: '@'

I have added isidentifier validation.